types/jquery/index.d.ts

```diff
@@ -22,7 +22,7 @@
 //                 Dick van den Brink <https://github.com/DickvdBrink>
 //                 Thomas Schulz <https://github.com/King2500>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.3
+// TypeScript Version: 2.8
```

Comment:
jQuery does not have a dependency on React. This version bump should not be necessary.

Reply:
react => react-mce => tinymce => jquery 😞 
Normally, I updated only packages where a dependency error was raised.
If you have a better solution, I take it 😉

Reply:
> react => react-mce => tinymce => jquery :disappointed:

This is the actual dependency graph. Neither `@types/jquery` or `@types/tinymce` should require the version bump.

![chrome_2018-03-28_14-57-17](https://user-images.githubusercontent.com/5265970/38050242-58ac9862-3298-11e8-9a17-67edd8009141.png)

Could you double check that jQuery indeed causes the error without the version bump?

It should be the case that only dependents of React need to be updated, not dependencies (@andy-ms ?). 

If it is the case that jQuery requires the version bump, I'd be against this change even when 2.8 becomes allowed on DefinitelyTyped. Increasing requirements soon after a new version becomes allowed seems to cause breaks for a lot of jQuery consumers. It'd be prudent to wait even longer to minimize this.

Reply:
Same as the comment above by @leonard-thieu with respect to `d3` et al. None of the D3 modules has an external dependency outside the D3 ecosystem, the only exception is `GeoJson` for a couple of module _definitions_. There is certainly no dependency on `react` or `react-faux-dom`.

As for our current versioning policy, we are migrating the definitions to start using some TS 2.3 features, as the release is ~1yr out and the installed based should be able to adopt readily. TS 2.8 is likely a while out, regardless of how beneficial the new features may be.

Reply:
Yes I confirm @leonard-thieu 
I understand your remarks.
This dependency system is blocking :/ Conditional types feature is sooo powerful to fix some issues.